.github/workflows/code_formatting_checks.yml

```diff
@@ -20,28 +20,35 @@ jobs:
         run: ./gradlew spotlessApply
 
       - name: Commit and Push
+        id: commit_and_push
         run: .github/scripts/commit_and_push.sh "${{secrets.USER_NAME}}" "${{secrets.USER_EMAIL}}"
 
       - name: Create PR
         uses: actions/github-script@0.3.0
+        env:
+          branch_name: ${{steps.commit_and_push.outputs.branch_name}}
         with:
           github-token: ${{secrets.GITHUB_TOKEN}}
           script: |
-            var monthNames = ["Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"];
-            var date = new Date();
-            var monthName = monthNames[date.getMonth()];
-            var day = `${date.getDate()}`.padStart(2, 0);
+            const monthNames = ["Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"];
+            const date = new Date();
+            const monthName = monthNames[date.getMonth()];
+            let day = `${date.getDate()}`.padStart(2, '0');
             day = `${monthName}-${day}-${date.getFullYear()}`
```

Comment:
Can we create separate variables to track the `day` and the `date`? This will allow us to make all this variables a `const`.

Reply:
Updated it. Please check now